cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+describe('Article Testing', () => {
+  // Assuming you need to log in before each test
+  beforeEach(() => {
+    // Visit the page
+    cy.visit('https://conduit.mate.academy/');
 
-  });
+    // Click "Sign In"
+    cy.get('a[href="/user/login"]').click();
 
-  it('', () => {
+    // Fill in email and password
+    cy.get('input[type="email"]').type('kotek@gmail.com');
+    cy.get('input[type="password"]').type('#4425ASt');
 
+    // Click the "Sign in" button
+    cy.get('button[type="submit"]').click();
   });
 
-  it('', () => {
+  it('should create and delete an article', () => {
+    // Click "New Article"
+    cy.get('a[href="/editor"]').click();
+
+    // Fill in article creation fields
+    cy.get('input[placeholder="Article Title"]').type('Test Article');
```

Comment:
You are supposed to use randomly generated data for an article creation. You need to add a new custom command `generateArticle()` to the config file, where the `generateUser()` function is located too.